certs/gen.go

```diff
@@ -8,14 +8,14 @@ import (
 	"math"
 	"sort"
 
+	big "math/big"
+
 	gpbft "github.com/filecoin-project/go-f3/gpbft"
 	cid "github.com/ipfs/go-cid"
 	cbg "github.com/whyrusleeping/cbor-gen"
-	xerrors "golang.org/x/xerrors"
-	big "math/big"
 )
 
-var _ = xerrors.Errorf
+var _ = fmt.Errorf
```

Comment:
Was this achieved by regenerating the code, or did you explicitly change it? If the cbor-gen tool uses xerrors, I think we should just accept it here rather than manually fix generated code.